Adds mutated monkeys for testing

comma opened the pull request:
Adds biohazard crates that are science-locked trash carts basically.
Adds spawnable empty vatgrown for when you need human testing but you just can't trick anyone into consenting.
Awake clientless mindless humans (only these blanks currently) will now bitch and moan when hurt.

Mostly done, just gotta map a couple of these somewhere in a nook with RD access.
![](http://i.imgur.com/3KyXR6z.png)

![](http://i.imgur.com/i2ZORqv.png)

comma committed:
Adds biohazard carts and test subjects

Mindless vatgrown wrapped in fabric and plastic.

mustafakalash reviewed (changes requested):
It looks like you've got the code from your assembly crate pr in here. I'm also going to say waiting lore, pending the debate on using sleeves for this. @SiegDerMaus

comma commented:
Yeah since it's based on that branch. Considering the speed of lore decisions I figure crate one will be merged way before.

babydoll commented:
How could we consider these things as mindless and subhuman if they show emotion and reaction to being harmed

comma commented:
So do animals, you don't need to be sapient to pull that off.
And  this isn't the place for that.
https://baystation12.net/forums/threads/how-about-dem-vat-blanks.4671/#post-69426

babydoll commented:
I mean you should just have these things be completely insensate

comma commented:
Then it would seem too clear cut :3
They come equipped with a muzzle for a reason.
Should probably add this to monkeys too actually.

comma committed:
Adjusts outfit equip order

Helmets can block masks.

comma committed:
Adds some NPC handling to humans and monkeys

They now react to the pain. Wouldn't trigger for those tha thave minds, e.g. logged off or ghosted.

Techhead0 commented on code/modules/mob/living/carbon/human/npcs.dm:
```diff
@@ -32,4 +32,31 @@
+	var/obj/item/clothing/under/color/white/C = locate() in H
+	if(C)
+		C.has_sensor = 2 //For the crew computer 2 = unable to change mode
+		C.sensor_mode = 0
```
These have defines. [See here.](https://github.com/Baystation12/Baystation12/blob/aa8e9875ae6d7d0fc983351fe46b48cae40c5e66/code/__defines/items_clothing.dm#L192-L199)

comma replied:
aight

mustafakalash reviewed (approved):
With Travis.

comma commented:
Travis Vult
